README.md

```diff
@@ -101,16 +112,20 @@ with SummaryWriter(logdir='./logs') as sw:
         sw.add_histogram(tag='norml_dist', values=data, bins=200, global_step=i)
 ```
 
-2. Run TensorBoard by typing the following command under the same directory as `logs`.
+**Step 2.** Run TensorBoard on the event logs.
+Use the following command to start the TensorBoard server. It will use the logs that were generated in the current directory's `logs` folder.
+
 ```bash
 tensorboard --logdir=./logs --host=127.0.0.1 --port=8888
 ```
 Note that in some situations,
 the port number `8888` may be occupied by other applications and launching TensorBoard
 may fail. You may choose a different available port number.
 
-3. In the browser, enter the address `127.0.0.1:8888`, and click the tab **HISTOGRAMS**
-in the TensorBoard GUI. You will see data distribution changing as time evolves.
+**Step 3.** Open TensorBoard in your browser.
+
+In the browser, enter the address `127.0.0.1:8888`, and click the tab **HISTOGRAMS**
+in the TensorBoard GUI. You will see data distribution changing as time progresses.
 ![png](https://github.com/reminisce/web-data/blob/tensorboard_doc/mxnet/tensorboard/doc/summary_histogram_norm.png)
```

Comment:
this doesn't preview normally... and should probably be in a more permanent location....

Reply:
Agree. I will ask Eric to merge the PR in the repo web-data and then we can link the pictures in the master/branch of that repo.